packages/priority-queue/src/request-idle-callback.js

```diff
@@ -2,7 +2,7 @@
  * @return {typeof window.requestIdleCallback|typeof window.requestAnimationFrame|((callback:(timestamp:number)=>void)=>void)}
  */
 export function createRequestIdleCallback() {
-	if ( typeof 'window' === undefined ) {
+	if ( typeof window === 'undefined' ) {
```

Comment:
Nice spotting. Would there by any chance of getting this in for `7.6.0` @aduth ?

Reply:
I can publish this fix to npm once merged.

Reply:
Interesting that `no-constant-condition` lint rule didn't catch this, I suppose that's because `undefined` can be a bound name?